src/room/usePTT.ts

```diff
@@ -30,6 +30,17 @@ function getActiveSpeakerFeed(
 ): CallFeed | null {
   const activeSpeakerFeeds = feeds.filter((f) => !f.isAudioMuted());
 
+  // make sure the feeds are in a deterministic order so every client picks
+  // the same one as the active speaker
+  const collator = new Intl.Collator("en", {
+    sensitivity: "variant",
+    usage: "sort",
+    ignorePunctuation: false,
+  });
+  activeSpeakerFeeds.sort((a: CallFeed, b: CallFeed): number =>
+    collator.compare(a.userId, b.userId)
+  );
```

Comment:
Why isn't `activeSpeakerFeeds.sort()` deterministic on its own?

Reply:
Nevermind, was blind to the fact that we're sorting by the `userId` key

Reply:
To be fair, this could be better commented - I'll add one.